normits_demand/models/notem/production_models.py

```diff
@@ -243,60 +252,52 @@ def run(self,
         # Initialise timing
         # TODO(BT): Properly integrate logging
         start_time = timing.current_milli_time()
-        du.print_w_toggle(
-            "Starting HB Production Model at: %s" % timing.get_datetime(),
-            verbose=verbose
-        )
+        self._logger.info("Starting a new run of NoTEM")
+        self._logger.info("Starting HB Production Model at: %s" % timing.get_datetime())
 
         # Generate the productions for each year
         for year in self.years:
             year_start_time = timing.current_milli_time()
-
-            # ## GENERATE PURE DEMAND ## #
-            du.print_w_toggle("Loading the population data...", verbose=verbose)
-            pop_dvec = self._read_land_use_data(year, verbose)
-
-            du.print_w_toggle("Applying trip rates...", verbose=verbose)
-            pure_demand = self._generate_productions(pop_dvec, verbose)
-
-            if export_pure_demand:
-                du.print_w_toggle("Exporting pure demand to disk...", verbose=verbose)
-                pure_demand.to_pickle(self.export_paths.pure_demand[year])
-
-            if export_reports:
-                du.print_w_toggle(
-                    "Exporting pure demand reports to disk...",
-                    verbose=verbose
-                )
-
-                report_seg = nd.get_segmentation_level('notem_hb_productions_pure_report')
-                pure_demand_paths = self.report_paths.pure_demand
-                pure_demand.aggregate(report_seg).write_sector_reports(
-                    segment_totals_path=pure_demand_paths.segment_total[year],
-                    ca_sector_path=pure_demand_paths.ca_sector[year],
-                    ie_sector_path=pure_demand_paths.ie_sector[year],
-                )
+        # ## GENERATE PURE DEMAND ## #
+        self._logger.info("Loading the population data...")
```

Comment:
You can probably remove the dots from here now that it's going into the log - repeat for all messages